src/aks-preview/azext_aks_preview/_loadbalancer.py

```diff
@@ -68,6 +69,14 @@ def configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, out
             profile.managed_outbound_i_ps = ManagedClusterLoadBalancerProfileManagedOutboundIPs(
                 count=managed_outbound_ip_count
```

Comment:
Should also add check for `managed_outbound_ip_count`. In an update scenario, user only specifies `managed_outbound_ip_count` this time, but we have `managed_outbound_ipv6_count` set when the cluster was created, this would overwrite `managed_outbound_ipv6_count` to `None`. Is this expected?

Reply:
yep, updated to preserve the ipv6 count through update commands.